src/chunk.c

```diff
@@ -2985,6 +3023,20 @@ ts_chunk_set_schema(Chunk *chunk, const char *newschema)
 	return chunk_update_form(&chunk->fd);
 }
 
+bool
+ts_chunk_add_status(Chunk *chunk, int status)
+{
+	return ts_chunk_set_status(chunk, chunk->fd.status | status);
```

Comment:
```suggestion
	return ts_chunk_set_status(chunk, ((uint32)chunk->fd.status) | (uint32)status);
```
Bitwise operators on signed operands produce implementation specific or undefined behaviour, so we must cast to unsigned first.